File: app/src/main/java/com/cheng/youthapartment/bean/UserBean.java
package com.cheng.youthapartment.bean;

import java.io.Serializable;

public class UserBean implements Serializable {
    private String nickname;
    private String avatarUrl;

    public UserBean() {

    }

    public UserBean(String nickname, String avatarUrl) {
        this.nickname = nickname;
        this.avatarUrl = avatarUrl;
    }

    public String getNickname() {
        return nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public void setAvatarUrl(String avatarUrl) {
        this.avatarUrl = avatarUrl;
    }

    @Override
    public String toString() {
        return "UserInfoBean{" +
                "nickname='" + nickname + '\'' +
                ", avatarUrl='" + avatarUrl + '\'' +
                '}';
    }
}

File: app/src/main/java/com/cheng/youthapartment/bean/user/UserBean.kt
package com.cheng.youthapartment.bean.user

/**
 *
 * @author Cheng
 * @since 2024/12/19
 */
data class UserBean(
    val nickname: String? = null,
    val avatarUrl: String? = null
) {
    override fun toString(): String {
        return "UserInfoBean{" +
                "\n          nickname = " + nickname +
                ",\n          avatarUrl = " + avatarUrl +
                "\n}"
    }
}